test/acceptance/behave/components/ngsiv2/subscriptions/create_a_new_subscription/create_a_new_subscription_subject_entities.feature

```diff
@@ -194,17 +235,17 @@ Feature: create new subscriptions (POST) using NGSI v2. "POST" - /v2/subscriptio
       # These properties below are used in subscriptions request
     And properties to subscriptions
-      | parameter               | value                                                                              |
-      | description             | "my first subscription"                                                            |
-      | subject_type            | "                   "                                                              |
-      | subject_idPattern       | ".*"                                                                               |
-      | condition_attributes    | "temperature"                                                                      |
-      | condition_expression    | "q">>>"temperature>40"&"georel">>>"near"&"geometry">>>"point"&"coords">>>"40.6391" |
-      | notification_callback   | "http://localhost:1234"                                                            |
-      | notification_attributes | "temperature"                                                                      |
-      | notification_throttling | 5                                                                                  |
-      | expires                 | "2016-04-05T14:00:00.00Z"                                                          |
-      | status                  | "active"                                                                           |
+      | parameter             | value                                                                                               |
+      | description           | "my first subscription"                                                                             |
+      | subject_type          | "                   "                                                                               |
+      | subject_idPattern     | ".*"                                                                                                |
+      | condition_attrs       | "temperature"                                                                                       |
+      | condition_expression  | "q">>>"temperature>40"&"georel">>>"near;MinDistance:1000"&"geometry">>>"point"&"coords">>>"40.6391" |
+      | notification_http_url | "http://localhost:1234"                                                                             |
+      | notification_attrs    | "temperature"                                                                                       |
+      | throttling            | 5                                                                                                   |
+      | expires               | "2016-04-05T14:00:00.00Z"                                                                           |
+      | status                | "active"                                                                                            |
     When create a new subscription in raw mode
```

Comment:
Ah, of course. I didn't see the pipe at end-of-line.
Now I scrolled and I've seen it.
My misunderstanding

NTC